README.md

```diff
@@ -49,18 +51,17 @@ Component | Spec | MDC Web
 
 > _NOTE:_ If you haven't used `create-react-app` before, you may want to read the [Overview Guide](https://github.com/facebook/create-react-app#quick-overview).
 
-
 #### Step 1: Install create-react-app
 
 Install `create-react-app`:
 
 ```
-npm i -g create-react-app
-create-react-app my-app
+npx create-react-app my-app
```

Comment:
This removed the step that actually installs create-react-app. I assume based on this it's intended to be installed locally instead of globally now?

Reply:
from what i read, `npx` installs a package globally. It is also what is on the create-react-app github page.


https://www.npmjs.com/package/npx

```
Executes <command> either from a local node_modules/.bin, or from a central cache, installing any packages needed in order for <command> to run.```